types/make.js

```diff
@@ -24,9 +24,25 @@ const base = fs.readFileSync(path.join(__dirname, 'base.d.ts'), 'utf8');
 // All suported function names
 const allParts = Object.keys(runner._chainableMethods).filter(name => name !== 'test');
 
-const output = base + generatePrefixed([]);
+// The output consists of the base declarations, the actual 'test' function declarations,
+// and the namespaced chainable methods.
+const output = [
+	base,
+	testFunctionDeclarations(),
+	generatePrefixed([])
+].join('\n');
+
 fs.writeFileSync(path.join(__dirname, 'generated.d.ts'), output);
 
+// Returns the declarations for the 'test' function and core types
+function testFunctionDeclarations() {
+	return (
+`export default test;
```

Comment:
That's not an issue, `base.d.ts` is indeed only used by the generation script to write `generated.d.ts`.